DFS/CourseScheduleII.swift

```diff
@@ -0,0 +1,63 @@
+/**
+ * Question Link: https://leetcode.com/problems/course-schedule/
+ * Primary idea: Kahn's Algorithms
+ * 1) Create the graph
+ * 2) Decorate each vertex with its in-degree
+ * 3) Create a set of all sources
+ * 4) While the set isn’t empty,
+ *     i. Remove a vertex from the set and add it to the sorted list
+ *     ii. For every edge from that vertex:
+ *         - Decrement the in-degree of the destination node
+ *         - Check all of its destination vertices and add them to the set if they have no incoming edges
+ * Time Complexity: O(|E| + |V|), Space Complexity: O(n^2)
+ * Recommand Reading: http://cs.brown.edu/courses/csci0160/lectures/14.pdf
+ */
```

Comment:
Thank you, merged. Could you please check the README as well? There are 3 parts need to be changed:

1. The number of questions we complete
2. DFS category
3. All questions list

Thank you, I will add you to the contribution list of README.